Sources/Service/Environment/Environment+DotEnv.swift

```diff
@@ -0,0 +1,65 @@
+
+import Core
+#if os(Linux)
+    import Glibc
+#else
+    import Darwin
+#endif
+
+extension Environment {
+    /// Loads the ".env" and the ".env.environment" files if they exist
+    public func loadDotEnv() {
```

Comment:
I think this could maybe do with `func load(env: String = ".env")` or similar instead of hardcoding it?